lib/modules/dosomething/dosomething_reportback/dosomething_reportback.drush.inc

```diff
@@ -104,3 +116,55 @@ function drush_dosomething_reportback_ds_reportback_run_nid_fix() {
   }
   print_r("All reportbacks updated \n");
 }
+
+/**
+ * Callback for batch-exclude-reportbacks command.
+ */
+function drush_dosomething_reportback_batch_exclude_reportbacks($nid, $run_nid, $quantity_limit) {
+    // Find all the fids that should be marked as excluded, based on params passed through. 
+    $fids_to_be_marked_as_excluded = db_query("SELECT rb.rbid, rb.nid, rb.run_nid, rb.quantity, rbf.status, rbf.fid, rbf.rbid
+                                               FROM dosomething_reportback rb
+                                               JOIN dosomething_reportback_file rbf on rb.rbid = rbf.rbid
+                                               WHERE rb.nid = :nid AND rb.run_nid = :run_nid AND rb.quantity < :quantity_limit AND rbf.status = 'pending'",
+                                               array(
+                                                ':nid' => $nid,
+                                                ':run_nid' => $run_nid,
+                                                ':quantity_limit' => $quantity_limit,
+                                               ));
+
+    // Change fids status to excluded 
+    foreach ($fids_to_be_marked_as_excluded as $fid) {
```

Comment:
the only other place this is in is in [this update](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_reportback/dosomething_reportback.install#L918-L922) and is a tiny bit different because it doesn't include the array with the variables. 

But yes, all the below should be the same! Is it possible to call the code from the hook? Or, I can refactor maybe to write that all into one function and just call the function in both places?